bot/cogs/geo_cog/streetviewrandomizer/street_view_random.py

```diff
@@ -0,0 +1,249 @@
+import bot.bot_secrets as bot_secrets
+import geopandas as gpd
+import logging
+import random
+from bot.cogs.geo_cog.streetviewrandomizer.coordinate import Coordinate
+from bot.cogs.geo_cog.streetviewrandomizer.countries import COUNTRIES
+from bot.cogs.geo_cog.streetviewrandomizer.street_view_static_api import StreetViewStaticApi
+from dataclasses import dataclass
+from io import BytesIO
+from math import isclose
+from PIL import Image
+from timeit import default_timer as timer
+
+
+"""
+StreetViewRandom: runs the functionality for retrieving randomized streetview from a set of coordinates.
+To view a detailed list of surveyed countries, 
+run the following command before the if-statement in find_available_image():
+
+    f'{country_df["ISO3"].values[0]} | lon: {random_lon:20} lat: {random_lat:20} | time: {elapsed_ms:8.2f}ms')
+"""
+API = StreetViewStaticApi()
+MAX_ATTEMPTS: int = 50
+IMG_PATH: str = f"bot/cogs/geo_cog/temp_assets/StreetView.jpg"
```

Comment:
```suggestion
IMG_PATH: str = "bot/cogs/geo_cog/temp_assets/StreetView.jpg"
```
No need for f-string.